src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -26,11 +28,22 @@
   "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" befindet sich bereits im Shaderpaket-Ordner!",
   "options.iris.shaders.disabled": "Shader: Deaktiviert",
   "options.iris.shaders.enabled": "Shader: Aktiviert",
+  "options.iris.back": "Zurück",
+  "options.iris.reset": "Zurücksetzen",
+  "options.iris.reset.tooltip": "Setze ALLE Einstellungen zurück und wende diese an?",
+  "options.iris.reset.tooltip.holdShift": "Halte SHIFT und klicke um zurückzusetzen",
```

Comment:
It's actually called UMSCHALTTASTE in German.

`Halte` also doesn't quite fit. `halte ... gedrückt` seems better in this case.

This also sounds a bit weird because you don't really have a subject in this question as well